com.unity.multiplayer.mlapi/Runtime/Core/NetworkObject.cs

```diff
@@ -22,29 +22,16 @@ public sealed class NetworkObject : MonoBehaviour
     {
         [HideInInspector]
         [SerializeField]
-        internal ulong GlobalObjectIdHash64;
+        internal ulong GlobalObjectIdHash;
```

Comment:
as per @TwoTenPvP's suggestion, we're dropping 64 suffix here and simplifying its name because `ulong` makes it obvious already.

Reply:
Do we want to leave a comment next to (above or to the side) that explains the data source that the hash was derived from?
// GlobalObjectIdHash is derived from the following pattern:  (PrefabPath) : (GlobalObjectId Guid)

Or...something to that nature?  
_Just thinking it could be useful to have that when trying to figure out how this new pattern works._

Reply:
it is not derived from {PrefabHash}:{GlobalObjectId} pattern, it's literally just the GlobalObjectId being hashed and that's it :)

Reply:
Got it!  I see the removal of the ValidateHash and the usage of the UnityEditor.GlobalObjectId.   
Makes sense!